function-service-impl/src/main/java/com/future/function/service/impl/feature/communication/chatroom/MessageStatusServiceImpl.java

```diff
@@ -121,13 +127,22 @@ public MessageStatus updateMessageStatus(
 
   @Override
   public void enterChatroom(String chatroomId, String userId) {
-    redisSetOperations.add("chatroom:" + chatroomId + ":active.user", userId);
+    if (chatroomId.equalsIgnoreCase(ChatroomType.PUBLIC.name())) {
+      chatroomId = chatroomService.getPublicChatroom().getId();
+    }
+    UriTemplate uriTemplate = new UriTemplate(redisProperties.getKey().get("active-chatroom"));
```

Comment:
Maybe initialize at constructor instead of initializing new object every time this method is called

Reply:
Ok, will be updated in next commit